module/Decision/src/Model/Member.php

```diff
@@ -708,26 +711,6 @@ public function toArray(): array
         ];
     }
 
-    /**
-     * @return array
-     */
-    public function toApiArray(): array
```

Comment:
Why remove this method? It seems like a usable utility function

Reply:
It was only used in the API that was used by REX. I have removed the API as REX does not longer "exist" and the endpoint is a privacy problem. Without the API there is no need to keep the function and the standard `toArray()` in `Member` is almost the same so that can be used instead.